scripts/ci-build-azure-ccm.sh

```diff
@@ -66,17 +66,17 @@ main() {
         echo "Building and pushing Linux and Windows amd64 Azure ACR credential provider"
         make -C "${AZURE_CLOUD_PROVIDER_ROOT}" bin/azure-acr-credential-provider bin/azure-acr-credential-provider.exe
 
-        if [[ "$(az storage container exists --name "${AZURE_BLOB_CONTAINER_NAME}" --query exists --output tsv)" == "false" ]]; then
+        if [[ "$(az storage container exists --name "${AZURE_BLOB_CONTAINER_NAME}" --query exists --output tsv --auth-mode login)" == "false" ]]; then
             echo "Creating ${AZURE_BLOB_CONTAINER_NAME} storage container"
-            az storage container create --name "${AZURE_BLOB_CONTAINER_NAME}" > /dev/null
+            az storage container create --name "${AZURE_BLOB_CONTAINER_NAME}" --auth-mode login > /dev/null
             # if the storage account has public access disabled at the account level this will return 404
-            az storage container set-permission --name "${AZURE_BLOB_CONTAINER_NAME}" --public-access container > /dev/null
+            az storage container set-permission --name "${AZURE_BLOB_CONTAINER_NAME}" --public-access container --auth-mode login > /dev/null
```

Comment:
this one won't work 😢 

```
az storage container set-permission --auth-mode login --name test --public-access container
az storage container set-permission: 'login' is not a valid value for '--auth-mode'. Allowed values: key
```

will need to set an env variable for this one

Reply:
`AZURE_STORAGE_AUTH_MODE` will work for this command

Reply:
Thanks! Fixed.